__main__.py

```
@@ -0,0 +1,607 @@
+        elif args.mode == 'exploit':
+            run_standard_mode(args)
+        elif args.mode == 'hunt':
+            run_autonomous_mode(args)
+        else:
+            parser.print_help()
+            sys.exit(1)
+
+    except KeyboardInterrupt:
+        print("\n[!] Interrupted by user")
+        sys.exit(130)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)